### ISSUE TITLE
Tablet UI/UX

### ISSUE THREAD

**Issue by sindastra** [vector-im/riotX-android#1843]
Currently, on the Android version of Element, you do not have a great experience on a tablet compared to the iPad version.

More specifically, a lot of space is wasted on landscape mode on a tablet. While on an iPad you do see your contacts and rooms on the left-hand side, the Android version simply shows the chat in full-screen with a small font which brings me to the next point: Increasing the default font-size on tablet in landscape mode. Please note that it should be made possible to close the left-hand side in case the user decides to have the chat in full-screen. Furthermore, I am missing the option to send by pressing enter on a physical keyboard.

### Recap

1. Show contacts and rooms on the left-hand side on tablet version (which can be optionally closed and re-opened)
2. Increase the font size on tablet in landscape mode
3. Allow for sending a message by pressing enter on a physical keyboard

In other words make better use of the big screen size and tablet form factor, like the iPad version of Element does.
